hmt_escrow_js/ETHInterface.js

```diff
@@ -0,0 +1,146 @@
+require('dotenv').config()
+
+const EscrowABI= require('./ABIs/Escrow')
+const EscrowFactoryABI = require('./ABIs/EscrowFactory')
+const HMTTokenInterfaceABI = require('./ABIs/HMTTokenInterface')
+
```

Comment:
You should fetch them from the artifacts, the ABI is subject to change, and on top of that, it shouldn't be uploaded. I think if you run `truffle compile` on the contract folder, the builded artifacts will be there, you should fetch them from there.

Reply:
Done.. check latest commits